opencti-platform/opencti-front/src/utils/tests/overviewLayoutCustomization.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { describe, it, expect } from 'vitest';
+import { renderHook } from '@testing-library/react';
+import React from 'react';
+import useAuth from '../hooks/useAuth';
+import { createMockUserContext, ProvidersWrapper, ProvidersWrapperProps } from './test-render';
+
+describe('overviewLayoutCustomization', () => {
+  const threatActorIndividualEntityType = 'Threat-Actor-Individual';
+  const overviewLayoutCustomization = new Map();
+  const threatActorIndividualConfiguration = new Map();
+  threatActorIndividualConfiguration.set('details', { order: 1, width: 6 });
```

Comment:
instead of doing a succession of 'set' you can directly write: newMap([[key1, value1], [key2, value2], ...]);